docs/user-guide/simultaneous_cosimulations.md

```diff
@@ -1,67 +1,2 @@
-# Simultaneous Co-simulations
```

Comment:
Was removing the rest of this guide the intended?

Reply:
I changed the name of the page to be better but I wanted the link to remain so this is a just a way to redirect.  All the content is now in webserver.md

Reply:
What about the sections on setting port numbers and use of keys?

Reply:
I think those are still relevant for multi-user systems like HPC nodes.

Reply:
You are correct I think I removed the wrong file,  I will correct that in the next commit